### PR TITLE
Fix intl

### LINKED ISSUE #150: React Intl object not found when opening new form
When I open new form I get following error:
image

### PR DESCRIPTION
**Pull request by LaChope**
Resolves #150

### CODE REVIEW

File: src/components/record/RecordForm.jsx [hunk 1]
```diff
@@ -116,16 +116,18 @@ class RecordForm extends React.Component {
...
-            ref={this.refForm}
-            form={this.props.form}
-            formData={this.props.record.question}
-            options={options}
-            fetchTypeAheadValues={this.fetchTypeAheadValues}
-            isFormValid={this.props.isFormValid}
-            enableForwardSkip={true}
-            loader={<Loader />}
-          />
+          <IntlContextProvider locale={options.intl.locale}>
```
Comment (LaChope) [inline src/components/record/RecordForm.jsx:119]: As mentioned in https://github.com/kbss-cvut/s-forms/issues/193, there are 2 ways of providing intl in SForms now, which should be refactored.

**Comment by blcham** [comment]
we wait to finish it systematically

**Comment by LaChope** [comment]
> we wait to finish it systematically

Relates to https://github.com/kbss-cvut/s-forms/pull/340

**Comment by blcham** [comment]
@LaChope this can be closed without merge right ? 

So what needs to be done to incorporate https://github.com/kbss-cvut/s-forms/pull/340 ?

**Comment by LaChope** [comment]
@blcham this can be closed because of https://github.com/kbss-cvut/s-forms/pull/340. When https://github.com/kbss-cvut/s-forms/pull/340 is merged, I will bump s-forms in record-manager-ui

**Pull request closed without merging**